containers/groovy-examples/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.4"
```

Comment:
@devinrsmith I see a bunch of repetition and some inconsistency here.  Do we want to keep things in single files, as they are, or use something like https://stackoverflow.com/questions/33408118/inheritance-or-nesting-with-docker-compose or https://serversforhackers.com/dockerized-app/compose-separated?  My concerns are DRY and inconsistencies.  I can see that avoiding these problems may increase the problems with users deploying, so the changes might not be worth it.

Reply:
Since we want the files to be downloadable, do we have a choice? Inheritance would mean having the user download multiple files, not just one.

Reply:
I"m not against single files if we can keep them consistent.